src/modules/organizations/containers/categories-container.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { setCurrentOrganization } from '../action-creators';
+import { organizationShape } from '../model';
+import CategoriesEditor from '../components/categories-editor';
+
+class CategoriesContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.resetOrganization = this.resetOrganization.bind(this);
+
+    this.state = {
+      categories: []
+    };
+  }
+
+  componentDidMount() {
+    this.setCategories();
+  }
+
+  setCategories() {
+    const categoriesWithKeys = this.props.organization.categories
+      .map(category => Object.assign({}, { key: Math.random(), category }));
+    this.setState({ categories: categoriesWithKeys });
+  }
+
+  handleSubmit() {
+    const categories = this.state.categories.map(categoryObject => categoryObject.category);
```

Comment:
Variable name choice: maybe `categoryObject.name` or `.title` instead of `categoryObject.category` would make it a bit more clear what this represents?